### PR TITLE
Overlay prototype

### PR DESCRIPTION
**Pull request by btd90**
Prototype for rendering images/videos on the map using imageOverlay and videoOverlay
Includes support for layer-control to allow toggling of overlays

### CODE REVIEW

File: app/objects/images/building-image.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import EmberObject from '@ember/object';
+
+/**
+ * Building image for Ember Leaflet.
+ */
+export default EmberObject.extend({
+  init() {
+    this._super(...arguments);
+    this.set('url', 'assets/images/building.png');
```
Comment (ultish) [inline app/objects/images/building-image.js:9]: cool, we'll be able to use attachment images too with a slight tweak :)

File: app/objects/videos/takeoff-video.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import EmberObject from '@ember/object';
+
+/**
+ * Takeoff video for Ember Leaflet.
+ */
+export default EmberObject.extend({
+  init() {
+    this._super(...arguments);
+    this.set('url', 'assets/images/takeoff.webm');
```
Comment (ultish) [inline app/objects/videos/takeoff-video.js:9]: any specific video encoding required?

File: app/services/draw-service.js [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+  /* Construct draw objects for draw-object component */
+  buildDrawObjects: function (newLayers) {
+    let builtObjects = A();
+
+    // Fetch each object
+    Object.keys(newLayers).forEach(key => {
+      let layer = newLayers[key];
+      if (layer._latlngs) {
+        if (layer._latlngs.length > 1) {
+          return builtObjects.pushObject({
```
Comment (ultish) [inline app/services/draw-service.js:24]: were the return statements in each of these conditions intentional for this forEach loop?

File: app/objects/videos/takeoff-video.js (hunk 2 above)
Comment (btd90) [reply app/objects/videos/takeoff-video.js:9]: There isn't much mention of specifics on the docs; I created the one with peek and it seems to run find
I'll test out an mp4 and report back

File: app/services/draw-service.js (hunk 3 above)
Comment (btd90) [reply app/services/draw-service.js:24]: good pickup, the final return statement should suffice

File: app/objects/videos/takeoff-video.js (hunk 2 above)
Comment (btd90) [reply app/objects/videos/takeoff-video.js:9]: peek records mp4 with h264 encoding, that also displays cleanly
I may do some trial and error on other encodings in the future

**Pull request merged**